src/main/java/edu/uc/inventorymanager/dto/Item.java

```diff
@@ -6,8 +6,8 @@
 
 @RequiredArgsConstructor
 public @Data class Item {
-    @NonNull private int id;
-    private String name;
+    @NonNull private int itemId;
+    private String itemName;
```

Comment:
Made variable names more specific so it is more clear what they are representing. This is a principle of clean code. Source: https://www.baeldung.com/java-clean-code

Reply:
I'd leave this as it were.
The context "Item" is a given, based on the class name, so adding "item" to each variable is redundant.